packages/jss-plugin-rule-value-function/src/index.js

```diff
@@ -22,7 +23,19 @@ export default function functionPlugin() {
     onCreateRule(name?: string, decl: JssStyle, options: RuleOptions): Rule | null {
       if (typeof decl !== 'function') return null
       const rule: StyleRuleWithRuleFunction = (createRule(name, {}, options): any)
-      rule[fnRuleNs] = decl
```

Comment:
this was how we hide the implementation detail of the interface that only this plugin knows how to handle